sdk/storage/storage-blob/test/blobbatch.spec.ts

```diff
@@ -273,14 +275,14 @@ describe("BlobBatch", () => {
     assert.ok(leaseResp.leaseId! != "");
 
     // Assemble batch set tier request.
-    let batchSetTierRequest = new BatchSetTierRequest();
-    await batchSetTierRequest.addSubRequest(blockBlobURLs[0], "Cool");
```

Comment:
maybe change one test to use the helper that takes list of urls? so we have test coverage for that.

Reply:
yeah I'm working on adding a test for each of those, just doing the record right now 👍